src/handlers/api.py

```diff
@@ -465,6 +481,26 @@ def write(self, chunk):
                     self.set_header('Content-Disposition', 'attachment; filename="smartapi_metakg.graphml"')
 
                 return super(BaseAPIHandler, self).write(chunk)
+            
+            if self.format == "html":
+                # reformat data
+                cdf = CytoscapeDataFormatter(chunk['hits'])
```

Comment:
@marcodarko need to handle possible errors here and other places below, when `hits` or `total` is not return (e.g. invalid query). In such cases, I think it would be reasonable to display in JSON mode with the actual response object.